core/src/verification/queue/mod.rs

```diff
@@ -429,6 +430,7 @@ impl<K: Kind> VerificationQueue<K> {
                 *td -= score;
             }
         }
+        processing.shrink_to_fit();
```

Comment:
This seems redundant.

Reply:
Yes, It's redundant. I fixed it. Thank you.